FogCreek/Glitch-Community
# Fifth reaction: Replace Default User & Team Cover Image

## Links
* Remix link: https://fifth-reaction.glitch.me/@scientiffic
* Specs/Notion docs: https://www.notion.so/glitch/Default-Cover-Images-6faf8b535e364ce6a05ba7443161011a

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/519336/59220670-7de31c00-8b93-11e9-9026-2b6dd821e80b.png)
![image](https://user-images.githubusercontent.com/519336/59220677-82a7d000-8b93-11e9-9afe-d2acfbf51e7b.png)

## Changes:
* Created cover image patterns for user and team pages
* The background color varies based on the user / team ID and selects from four pre-set light colors. `lightColors = ['#FFC761', '#9BFFB5', '#FF9BA1', '#9B9EFF'];` 

## How To Test:
* Check out team pages that have the default cover image, such as https://fifth-reaction.glitch.me/@community
* Check out user pages that have the default cover image, such as https://fifth-reaction.glitch.me/@scientiffic
* Check out search results to see corresponding cover color for users / teams that haven't overridden default https://fifth-reaction.glitch.me/search?q=hello

## Feedback I'm looking for:
* Currently I'm storing `lightColors` in `/Models/user` but maybe there's a better central location to put it.
